validation-framework/build.sh

```diff
@@ -10,16 +10,30 @@ chartver=""
 if [ "$2" ];then
   chartver="$2"
 fi
-
+composeCMD="docker-compose"
+containerDelim="-"
 opt3=$3
-
+function compose {
+  if [ "$composeCMD" = "podman-compose" ]; then
+    podman-compose $@
+  else
+    docker-compose $@
```

Comment:
should we choose "docker compose" over docker-compose?
https://docs.docker.com/compose/migrate/

Reply:
updated